ZenPacks/zenoss/WBEM/modeler/wbem.py

```diff
@@ -92,8 +120,10 @@ def collect(self, device, log):
                     userCreds, namespace=namespace)
 
             elif wbemclass == 'ei':
-                wbemClass = EnumerateInstances(
-                    userCreds, namespace=namespace, classname=classname)
+                wbemClass = getEnumerateInstances(
+                    userCreds, namespace=namespace, classname=classname,
+                    MaxObjectCount=device.zWBEMMaxObjectCount,
```

Comment:
The first letter for MaxObjectCount argument must be in lower case.

Reply:
we can not change `MaxObjectCount` because we use this name to create a correct request to the target system.